lib/ret/uploads.ex

```diff
@@ -18,7 +18,6 @@ defmodule Ret.Uploads do
           meta = %{
             content_type: content_type,
             content_length: content_length,
-            filename: filename,
```

Comment:
Oh, I guess you could also remove it from the pattern match in this method's signature?

Reply:
ah yep, thanks!